### PR TITLE
Add initial Sentiment Analysis Tutorial files for Cloud NL API

### PR DESCRIPTION
**Pull request by manshreck**
Change includes tutorial code, test code, and test review files

### CODE REVIEW

File: language/sentiment/gcloud/.Python [hunk 1]
```diff
@@ -0,0 +1 @@
+/Library/Frameworks/Python.framework/Versions/2.7/Python
```
Comment (jerjou) [inline language/sentiment/gcloud/.Python]: Can you remove the gcloud dir from the checkin?

File: language/sentiment/neg.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+What was Hollywood thinking with this movie! I hated,
```
Comment (jerjou) [inline language/sentiment/neg.txt]: Can you rename the `sentiment` dir to `resources`, for consistency with the other samples

File: language/sentiment/out [hunk 3]
```diff
@@ -0,0 +1 @@
...bG9zb3BoaWNhbApleHBsb3JhdGlvbiBvZiB3aGF0IGl0IG1lYW5zIHRvIGJlIGh1bWFuIG9yIG1hY2hpbmUuIFNvbWUgb2YgdGhlIGdpem1vcwpsaWtlIHRoZSBmbHlpbmcgY2FycyBhbmQgdGhlIFZvdWlnaHQtS2FtcGZmIG1hY2hpbmUgKHdoaWNoIHNlZW1lZCB2ZXJ5CnN0ZWFtcHVuayksIHdlcmUgcXVpdGUgY29vbC4KCkkgZGlkIGZpbmQgdGhlIHBsb3QgcHJldHR5IHNsb3cgYW5kIGJ1dCB0aGUgZGlhbG9ndWUgYW5kIGFjdGlvbiBzZXF1ZW5jZXMKd2VyZSBnb29kLiBVbmxpa2UgbW9zdCBzY2llbmNlIGZpY3Rpb24gZmlsbXMsIHRoaXMgb25lIHdhcyBtb3N0bHkgcXVpZXQsIGFuZApub3QgYWxsIHRoYXQgbXVjaCBoYXBwZW5lZCwgZXhjZXB0IGR1cmluZyB0aGUgbGFzdCAxNSBtaW51dGVzLiBJIGRpZG4ndAp1bmRlcnN0YW5kIHdoeSBhIHVuaWNvcm4gd2FzIGluIHRoZSBtb3ZpZS4gVGhlIHZpc3VhbCBlZmZlY3RzIHdlcmUgZmFudGFzdGljLApob3dldmVyLCBhbmQgdGhlIG11c2ljYWwgc2NvcmUgYW5kIG92ZXJhbGwgbW9vZCB3YXMgcXVpdGUgaW50ZXJlc3RpbmcuCkEgZnV0dXJpc3QgTG9zIEFuZ2VsZXMgdGhhdCB3YXMgYm90aCBoaWdobHkgcG9saXNoZWQgYW5kIGFsc28gZmFsbGluZyBhcGFydApyZW1pbmRlZCBtZSBvZiAnT3V0bGFuZC4nIENlcnRhaW5seSwgdGhlIHN0eWxlIG9mIHRoZSBmaWxtIG1hZGUgdXAgZm9yCm1hbnkgb2YgaXRzIHBlZGFudGljIHBsb3QgaG9sZXMuCgpJZiB5b3Ugd2FudCB5b3VyIHNjaS1maSB0byBiZSBsYXNlcnMgYW5kIHNwYWNlc2hpcHMsICdCbGFkZXJ1bm5lcicgbWF5CmRpc2FwcG9pbnQgeW91LiBCdXQgaWYgeW91IHdhbnQgaXQgdG8gbWFrZSB5b3UgdGhpbmssIHRoaXMgbW92aWUgbWF5CmJlIHdvcnRoIHRoZSBtb25leS4KCg==
```
Comment (jerjou) [inline language/sentiment/out]: I think this file isn't necessary?

File: language/sentiment/sentiment_analysis.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+def main(movie_review_filename):
+    '''Run a sentiment analysis request on text within a passed filename.'''
+
+    credentials = GoogleCredentials.get_application_default().create_scoped(
+        ['https://www.googleapis.com/auth/cloud-platform'])
+    http = httplib2.Http()
+    credentials.authorize(http)
+
+    service = discovery.build('language', 'v1beta1', http=http)
```
Comment (jerjou) [inline language/sentiment/sentiment_analysis.py]: You can just do:

```py
credentials = GoogleCredentials.get_application_default()
service = discovery.build('language', 'v1beta1', credentials=credentials)
```

The python client lib will take care of adding the right scope, and authorizing the http object.

File: language/sentiment/sentiment_analysis.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                }
+            }
+        )
+        response = service_request.execute()
+
+    try:
+        polarity = response['documentSentiment']['polarity']
+        magnitude = response['documentSentiment']['magnitude']
+    except KeyError:
+        print("The response did not contain the expected fields.")
```
Comment (jerjou) [inline language/sentiment/sentiment_analysis.py]: Just let the `KeyError` propagate up. Error handling doesn't really provide much educational value IMO..

**Comment by jonparrott** [comment]
Hey, whoa, please don't merge without review! I'm going to revert this - please open another PR.

File: language/sentiment/sentiment_analysis.py [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    print('Sentiment: polarity of %s with magnitude of %s'
+          % (polarity, magnitude))
+    return 0
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
```
Comment (jerjou) [inline language/sentiment/sentiment_analysis.py]: Add a module-level docstring and set that as the `description`. See <http://go/python-sample-guide#argparse-section>

File: language/sentiment/sentiment_analysis.py [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        )
+        response = service_request.execute()
+
+    try:
+        polarity = response['documentSentiment']['polarity']
+        magnitude = response['documentSentiment']['magnitude']
+    except KeyError:
+        print("The response did not contain the expected fields.")
+    print('Sentiment: polarity of %s with magnitude of %s'
+          % (polarity, magnitude))
```
Comment (jerjou) [inline language/sentiment/sentiment_analysis.py]: Prefer using the `'string {}'.format(arg)` syntax

File: language/sentiment/sentiment_analysis.py (hunk 5 above)
Comment (jerjou) [inline language/sentiment/sentiment_analysis.py]: style: prefer single quotes over double quotes

**Pull request merged**